### ISSUE TITLE
Lag on Jnet

### ISSUE THREAD

**Issue by BurntRope** [mtgred/netrunner#7584]
Hi, over the last few days game play is experiencing significant lag. Often experienced as a delay between a player clicking an action and the action running in the game. As well as having to click the same action multiple times (e.g. Run 'server') and then nothing happening in the game.

**Comment by NBKelly** [comment]
Just because it looks bad having no reply:
* Yes, we are aware about the lag
* The causes of this lag are pretty complex (there are a bunch of confounding factors that have built up over the last decade), and we're doing our best to determine what's actually causing the lag
* I believe we've made some significant improvements over the last few releases, but there's still both more to be done and a lot more to be investigated
* We can kind of simulate server load on the backend, but there are more circumstances in play than just server load
TLDR is we're working on it, but the tooling we need to accurately diagnose the issues isn't there, and we need to experiment, investigate, and invent it as we go.

**Comment by NBKelly** [comment]
I think we're at a point where this can be closed (at least until the next time we break everything). Games are running silk smooth for now.

If we ever get consistently bad again, feel free to create another issue.
